packages/core/src/components/Table/Body/Cell/ExtendedRowCell/ExtendedRowCell.tsx

```diff
@@ -17,6 +17,12 @@ export const ExtendedRowCell: React.FC<ExtendedRowCellProps> & WithStyle = React
         expandedRowComponent: ExpandedRowComponent
     } = props;
     const stopPropagation = useCallback((e: React.MouseEvent) => e.stopPropagation(), []);
+    const VisibleExpandedRow: React.SFC = props => {
```

Comment:
@AbhijitHycinth Why do we need to create a new FC for the case of just not rendering the component ???

What was the problem in using below code ??

```
{isRowExpanded && (<Wrapper tableSize={tableSize}>
        <ExpandedRowComponent rowId={rowId} rowData={rowData} disabled={isRowClickDisabled} />
</Wrapper>)}
```

I really don't like the idea of creating a new component for a condition.